addons/l10n_ec_stock/models/account_chart_template.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    def _l10n_ec_setup_location_accounts(self, companies):
+        for company in companies:
+            parent_location = self.env.ref('stock.stock_location_locations_virtual', raise_if_not_found=False)
+            loss_loc = parent_location.child_ids.filtered(lambda l: l.usage == 'inventory' and l.company_id == company and l.scrap_location is False)
```

Comment:
`raise_if_not_found=False` will return `None` if not found or an empty recordset? If the first then shouldn't this case be handled?